emacs.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
are we publishing the file format for this somewhere?  i.e. what are all of the available bindings?

seems like it should be in this repo or at least linked to

Reply:
We can, however I'm not sure of the best way to present it. The available actions are tied to our internal implementation and will continue to grow as we add new actions. Maintaining a full list of available actions will add some ongoing maintenance burden.

Reply:
Maybe, we can just loop this under documentation and I can manage keeping it up to date.

Reply:
Updated to include a `FORMAT.md` file that describes the file format and lists the available actions.